playbooks/roles/discourse/tasks/main.yml

```diff
@@ -0,0 +1,76 @@
+---
+- name: mkdir /discourse
+  file:
+    path: /discourse
+    state: directory
+
+- name: mount /discourse
+  mount:
+    name: /discourse
+    src: /dev/vdb1
+    fstype: ext4
+    state: mounted
+
+- name: mkdir /discourse/discourse (needed to have /var/discourse as a git repo, which can't use a mounted folder)
+  file:
+    path: /discourse/discourse
+    state: directory
+
+- name: ln -s /discourse/discourse /var/discourse
+  file:
+    src: /discourse/discourse
+    dest: /var/discourse
+    state: link
+
+- name: mkdir /discourse/docker
+  file:
+    path: /discourse/docker
+    state: directory
+
+- name: change docker root directory
+  lineinfile:
+    path: /lib/systemd/system/docker.service
+    regexp: '^ExecStart='
+    line: ExecStart=/usr/bin/dockerd -H fd:// --containerd=/run/containerd/containerd.sock -g /discourse/docker
```

Comment:
I feel like this should be a service override file, rather than changing the service inline?  eg. `/etc/systemd/system/docker.service.d/override-docker-root-dir.conf`

Reply:
Not high priority

Reply:
@swalladge You are totally right. I have changed the way to overwrite Docker settings by using `/etc/docker/daemon.json` instead.

Reply:
Ah, even better! :)